apps/flame-defi/app/constants.ts

```diff
@@ -15,6 +15,11 @@ export enum TXN_STATUS {
   FAILED = "failed",
 }
 
+export const flameExplorerUrl = "https://explorer.flame.astria.org";
+
+export const defaultApprovalAmount =
```

Comment:
I think I would prefer to keep this sort of thing in `AppConfig` where the rest of our config is. Let's prefix the name to make it more clear what this is being used for, like `TokenDefaultApprovalAmount`